app/src/main/java/agency/five/codebase/android/movieapp/model/Movie.kt

```diff
@@ -5,5 +5,5 @@ data class Movie(
     val title: String,
     val overview: String,
     val imageUrl: String?,
-    val isFavorite: Boolean,
+    var isFavorite: Boolean,
```

Comment:
I understand the idea but this is bad practice. Kotlin adopts a lot of principles of functional programming which is based on immutability. If your variables are immutable, then there are no concurrent problems :D
Because of this our models always have immutable properties (all vals) and when we want to mutate property we create a new object instead. 
Please return this to val .